pkg/crc/preflight/preflight_darwin.go

```diff
@@ -40,7 +40,7 @@ var dnsPreflightChecks = [...]PreflightCheck{
 }
 
 var traySetupCheck = PreflightCheck{
-	configKeySuffix:  "check-tray-setup",
+	configKeySuffix:  "",
```

Comment:
why remove this?

Reply:
It doesn't have any use now, since we are controlling the check's happening through the flag? so if i do `crc setup --enable-optional-features` and that config is set to `true` the setup will be skipped. and in the case of just `crc setup` that flag is not at all being used.